utils/releaser/tests/FileManipulator/UpgradeFileManipulator/Source/UPGRADE-after.md

```diff
@@ -0,0 +1,11 @@
+# Upgrade
+
+## [From v1.0.0 to Unreleased]
+
+## [From v0.9.0 to v1.0.0]
```

Comment:
maybe here is problem because we do not use `v` but we just start with numbers in .md please see alpha versions or less than beta3 if this is wrong from the view of OSS please contact @vitek-rostislav

Reply:
yes, let us discuss it :slightly_smiling_face: